### DISCUSSION TITLE
Pricing for using mapbox with custom tile servers

### DISCUSSION THREAD

**Discussion by srilekhachowdaryananthaneni** [rnmapbox/maps#1352] [Q&A] [answered]
I am working on a commercial product(react native mobile application supporting Android and iOS) where we need to display a map view on a screen. We want to use a custom tile server(eg: OpenStreetMaps) as a data source without using Mapbox servers. It seems we need to anyway set access token even though custom tile servers are used. We would want to know if we will be billed in case we use custom servers without using Mapbox servers?
Please let me know if we would be billed while using Mapbox with the above-mentioned conditions.

**Comment by ferdicus** [comment, marked as answer]
hi, if you are using the default native SDKs that come packed in with our repo, you only need an `accessToken` if you want to use mapbox tiles. 
So in the case you mentioned that would work out fine.
IF you want to use newer SDK versions, you'll at the very least need a download api token - check the installation guide of [ios](https://docs.mapbox.com/ios/maps/guides/install/#configure-credentials) and [android](https://docs.mapbox.com/android/maps/guides/install/#configure-credentials) for more info. 

Via this download mechanism (I think, don't quote me on this), mapbox checks the number of used seats/ active users.
This is currently set to 25000.

If you prefer an opensource flavor of mapbbox, check out installation guide to set up MapLibre :)

**Reply by mzu** [reply]
Looks like [mapbox](https://www.mapbox.com/pricing/#session-user-pricing) is using the term **seat** only for ([logged-in?](https://www.reddit.com/r/mapbox/comments/f8jody/what_does_map_seats_for_web_means/)) web users. For the sdk an active user identified by the `accessToken` is counted once per month in regards to billing. As you said, free limit for the sdk is set to 25k active users per month. All slightly confusing.
